presto-vertica/src/test/java/io/prestosql/plugin/vertica/TestVerticaIntegrationSmokeTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.vertica;
+
+import io.prestosql.testing.AbstractTestIntegrationSmokeTest;
+import io.prestosql.testing.MaterializedResult;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.testing.sql.SqlExecutor;
+import io.prestosql.testing.sql.TestTable;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static io.prestosql.plugin.vertica.TestingVerticaServer.SCHEMA;
+import static io.prestosql.plugin.vertica.VerticaQueryRunner.createVerticaQueryRunner;
+import static io.prestosql.spi.type.VarcharType.VARCHAR;
+import static io.prestosql.testing.assertions.Assert.assertEquals;
+import static io.prestosql.tpch.TpchTable.CUSTOMER;
+import static io.prestosql.tpch.TpchTable.NATION;
+import static io.prestosql.tpch.TpchTable.ORDERS;
+import static io.prestosql.tpch.TpchTable.REGION;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestVerticaIntegrationSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    protected TestingVerticaServer verticaServer;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        verticaServer = new TestingVerticaServer();
+        return createVerticaQueryRunner(verticaServer, CUSTOMER, NATION, ORDERS, REGION);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public final void destroy()
+    {
+        verticaServer.close();
+    }
+
+    @Test
+    @Override
+    public void testDescribeTable()
+    {
+        // Replaced shippriority column's 'integer' with 'bigint'
```

Comment:
make it declarative

```suggestion
        // shippriority column's type is BIGINT, not INTEGER
```

(same below)